zuul.d/jobs.yaml

```diff
@@ -478,3 +478,48 @@
     run: playbooks/packer-functional-devstack/run.yaml
     vars:
       packer_version: 1.2.5
+
+- job:
+    name: docker-machine-master-functional-opentelekomcloud
+    parent: init-test
+    description: |
+      Test functionalities of Docker machine of master against opentelekomcloud.
+    run: playbooks/docker-machine-functional-public-clouds/run.yaml
+    vars:
+      cloud_name: opentelekomcloud
+    secrets:
+      - opentelekomcloud_credentials
+
+- job:
+    name: docker-machine-maste-functional-orange
+    parent: init-test
+    description: |
+      Test functionalities of Docker machine of master against orange.
+    run: playbooks/docker-machine-functional-public-clouds/run.yaml
+    vars:
+      cloud_name: orange
+    secrets:
+      - orange_credentials
+
+- job:
+    name: docker-machine-maste-functional-telefonica
+    parent: init-test
+    description: |
+      Test functionalities of Docker machine of master against telefonica.
+    run: playbooks/docker-machine-functional-public-clouds/run.yaml
+    vars:
+      cloud_name: telefonica
+      ssh_user: linux
```

Comment:
If we use image `Ubuntu 16.04 Server 64bit` that we upload for all of public cloud, `ssh_user` should be default value `ubuntu`

Reply:
IN telefonica, the `Ubuntu 16.04 Server 64bit` is not uploaded by us, and Telefonica don't support upload .img image using openstack command

Reply:
Thank your clarification, it's necessary.